datalad/support/external_versions.py

```diff
@@ -252,6 +255,43 @@ def keys(self):
     def __contains__(self, item):
         return bool(self[item])
 
+    def add(self, name, func=None, interesting=True):
+        """Add a version checker
+
+        Could be used e.g. by extensions to add important to them custom checks.
+        If will not add `name` if already exists.  If it exists and `func` is
+        different - it will override with a new `func`.
+
+        Parameters
+        ----------
+        name: str
+          Name of the check (usually a name of the Python module, or an
+          external command prefixed with "cmd:")
+        func: callable, optional
+          Function to be called to obtain version information, if it is not
+          a Python module following standard ways to expose version via
+          `__version__`
```

Comment:
This description could be read as saying that `func` won't be called if `name` is not a Python module with `__version__`.  Perhaps "Function to be called to obtain version information.  This should be defined when checking the version of something that is not a Python module or when this class's method for determining the version of a Python module isn't sufficient."